src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,125 @@
+package duke;
+
+import java.io.File;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+
+public class Parser {
+
+    /**
+     * Method that adds a Deadline task onto TaskList
+     *
+     * @param input The string that contains the user input
+     * @param t The TaskList that contains the tasks to be added
```

Comment:
perhaps the naming of some of the parameters could be more verbose; however this is just nitpicky, since your javadocs already explain what the parameters mean. Good work! 👍🏻